### PR TITLE
{bio}[gompi/2021a] BGC-Bayesian-genomic-clines v1.03

### PR DESCRIPTION
**Pull request by stevenvdb**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/verdurin/07370dbd4e272cd7523df32c8ee11801 for a full test report.

**Comment by verdurin** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/b/BGC-Bayesian-genomic-clines/BGC-Bayesian-genomic-clines-1.03-gompi-2021a.eb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+source_urls = [GITHUB_SOURCE]
+sources = ['%s.tar.gz' % local_commit]
+patches = [('%(name)s-%(version)s_Makefile.patch', 1)]
+checksums = [
+    {'9c8c8cb.tar.gz': '105cd1f545c5d9cd55d965436e8e31934fc77a5981399aa6373acd6e42f73cf1'},
+    {'BGC-Bayesian-genomic-clines-1.03_Makefile.patch':
+     'c32853077658731dba2ac7f4916a67f43c9f7679916d602750a8c70d60d810de'},
+]
+
+dependencies = [('GSL', '2.7'),
```
Comment (verdurin) [inline easybuild/easyconfigs/b/BGC-Bayesian-genomic-clines/BGC-Bayesian-genomic-clines-1.03-gompi-2021a.eb:22]:
```suggestion
dependencies = [
    ('GSL', '2.7'),
```

File: easybuild/easyconfigs/b/BGC-Bayesian-genomic-clines/BGC-Bayesian-genomic-clines-1.03-gompi-2021a.eb [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+                ('HDF5', '1.10.7')]
```
Comment (verdurin) [inline easybuild/easyconfigs/b/BGC-Bayesian-genomic-clines/BGC-Bayesian-genomic-clines-1.03-gompi-2021a.eb:23]:
```suggestion
    ('HDF5', '1.10.7')
]
```

Comment (verdurin) [reply easybuild/easyconfigs/b/BGC-Bayesian-genomic-clines/BGC-Bayesian-genomic-clines-1.03-gompi-2021a.eb:23]: Bit nitpicky, I know, but I think that's the normal style we use for dependencies.

**Review by verdurin — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/b/BGC-Bayesian-genomic-clines/BGC-Bayesian-genomic-clines-1.03-gompi-2021a.eb (hunk 2 above)
Comment (stevenvdb) [reply easybuild/easyconfigs/b/BGC-Bayesian-genomic-clines/BGC-Bayesian-genomic-clines-1.03-gompi-2021a.eb]: No problem, updated in d17c806713b1471aca6cf219219ec4427465144d

**Review by verdurin — APPROVED** [review]
Looks fine.

**Comment by verdurin** [comment]
Going in, thanks @stevenvdb!

**Pull request merged**